docs/sf-methods.Rmd

```diff
@@ -0,0 +1,108 @@
+
+The above definition captures our mechanistic assumptions for the expectation of secondary notifications but does not account for potential observation noise or reporting patterns.
+Here we assume a negative binomial observation model (though our implementation also supports a Poisson observation model) in order to capture potential reporting overdispersion $\phi$ and adjust expected counts using an optional day of the week effect based on a simplex $\omega_{(t \mod 7)}$ (such that $\sum^6_{t=0}{w_t} = 7$ so the total effect over a week is balanced).
+This gives the following observation process. 
+
+\begin{equation} 
+    S_{t} \sim \mathrm{NB}\left(\omega_{t \mod 7} \hat{S}_t,  \phi \right)
+\end{equation}
+
+# Model priors
+
+We define weakly informative default priors for all parameters based on subject area knowledge. These are as follows,
+
+\begin{align}
+    \frac{\omega_{t \mod 7}}{7} &\sim \mathrm{Dirichlet}(1, 1, 1, 1, 1, 1, 1) \\
+    \alpha &\sim \mathcal{N}(0.5, 0.25) \\
+    \mu &\sim \mathcal{N}(2.19, 0.5) \\
+\sigma &\sim \mathcal{N}(0.47, 0.25) \\
+\phi &\sim \frac{1}{\sqrt{\mathcal{N}(0, 1)}}
+\end{align}
+
+Note that we choose $\mu$ and $\sigma$ such that the discretised log normal distribution has a mean of 10 and a standard deviation of 5 on the natural scale. We also set maximum delay $D$ to 30 for computational tractibility. Finally, $\alpha$, $\sigma$, and $\phi$ are truncated to be greater than 0.
+
+# Defining a surveillance indicator
+
+We now use the convolution model described in the last section to construct an indicator for whether or not the relationship between primary and secondary notifcations is changing over time. 
+We do this using a counterfactual approach by first fitting the model to a baseline period of time and then projecting this forward to the time period of interest. 
+Under an assumption of no change this approach should provide reasonable predictive accuracy and if it does not we may be able to conclude that there has been an underlying change in the relationship between the two indicators of interest.
+In order to account for the expected in sample performance of our model (potential misspecification etc.) we normalise performance of predictions from our baseline time period using predictions from a model fit to the time period of interest. 
+This then provides a relative metric of how well the model that assumes no change in indicator relationship captures the current relationship. 
+More detail is provided in the following sections.
+
+## Baseline time window versus the time window of interest
+
+We first fit a model to an initial time window which is by default from $w_b + w_t$ (where $w_b$ is the length of the baseline window and $w_t$ is the length of the target window of interest) days before the target date of interest until $w_t$ days before the target date of interest. 
```

Comment:
I've read this a few times and still don't get it. Do you mean we fit the model with and without the window of interest?

Reply:
I see now. Thanks. I have no suggested rewording.